.github/workflows/release.yml

```diff
@@ -34,14 +34,22 @@ jobs:
         with:
           python-version: "3.9"
 
-      - name: Install dependencies
+      - name: install build requirements
         run: |
-          pip install -r dev-requirements.txt
+          pip install build
           pip freeze
 
-      - name: Build distribution archives
+      - name: build release
         run: |
-          python setup.py sdist bdist_wheel
+          python -m build --sdist --wheel .
+          ls -l dist
+
+      # ref: https://github.com/actions/upload-artifact#readme
+      - uses: actions/upload-artifact@v3
+        with:
+          name: repo2docker_service-${{ github.sha }}
```

Comment:
Curios, why `_service` instead of just `repo2docker-`?

Reply:
Copy paste mistake of mine related to work on providing a jupyterhub service exposing repo2docker functionality! Nice catch